plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/part/DBeaverNavigationKeyHandler.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.erd.ui.part;
+
+import org.eclipse.gef3.GraphicalViewer;
+import org.eclipse.gef3.ui.parts.GraphicalViewerKeyHandler;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.KeyEvent;
+
+public class DBeaverNavigationKeyHandler extends GraphicalViewerKeyHandler {
+    public DBeaverNavigationKeyHandler(GraphicalViewer viewer) {
+        super(viewer);
+    }
+
+    boolean additionalAcceptIntoContainer(KeyEvent event) {
+        return event.keyCode == SWT.CR;
+    }
+
+    boolean additionalOutOfContainer(KeyEvent event) {
+        return event.keyCode == 8;
```

Comment:
Could you add a comment about what the 8 is? Or maybe it should be a constant...

Reply:
Is it Backspace?